data/plugins.json

```diff
@@ -88,15 +90,18 @@
     "node": 5
   },
   "transform-es2015-sticky-regex": {
-    "chrome": 49,
-    "edge": 13,
-    "firefox": 3,
-    "safari": 10,
-    "node": 6
+    "chrome": -1,
```

Comment:
This one is interesting since it seems as if the transform doesn't actually work?